function foo() {
  try {

  } catch (error) {

  }
}
